libs/@local/graph/postgres-store/src/store/postgres/knowledge/entity/mod.rs

```diff
@@ -411,359 +372,288 @@ where
         Ok(())
     }
 
-    #[tracing::instrument(level = "info", skip(self, params))]
-    async fn get_entities_impl(
+    #[tracing::instrument(level = "info", skip(self, params, entity_provider))]
+    async fn get_entities_impl<S: Sync>(
         &self,
         actor_id: ActorEntityUuid,
-        mut params: GetEntitiesImplParams<'_>,
+        entity_provider: &StoreProvider<'_, S>,
+        params: GetEntitiesImplParams<'_>,
         temporal_axes: &QueryTemporalAxes,
-    ) -> Result<(GetEntitiesResponse<'static>, Zookie<'static>), Report<QueryError>> {
-        let mut root_entities = Vec::new();
-        let filters: [Filter<'_, Entity>; 1] = [params.filter];
+    ) -> Result<GetEntitiesResponse<'static>, Report<QueryError>> {
+        let mut filters = vec![params.filter];
 
-        let (
-            permissions,
-            count,
-            web_ids,
-            created_by_ids,
-            edition_created_by_ids,
-            type_ids,
-            type_titles,
-        ) = if params.include_count
-            || params.include_web_ids
-            || params.include_created_by_ids
-            || params.include_edition_created_by_ids
-            || params.include_type_ids
+        if let Some((actor, _consistency, ref policy_set, ref policy_context)) =
```

Comment:
will we make use of the consistency at a later date?

Reply:
consistency is used somewhere else, so the comment is moot

Reply:
Not anymore, no,  but I didn't want to create a new struct which I will remove in a follow-up just to save a few destructured fields. And I preferred `_consistency` over `_` to at least see what's going on of inline hints are not available.